test/units/testsuite-99.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -eux
+set -o pipefail
+
+at_exit() {
+    # "Safety net" - check for any coredumps which might have not caused dfuzzer
+    # to stop & return an error (we need to do this now before truncating the
+    # journal)
+    while read -r exe; do
+        coredumctl info "$exe"
+    done < <(coredumpctl -F COREDUMP_EXE | sort -u)
+
+    # Limit the maximum journal size (because fuzzing)
+    journalctl --rotate --vacuum-size=16M
+}
+
+trap at_exit EXIT
+
+systemctl log-level info
+
+# TODO
+#   * check for possibly newly introduced buses?
+BUS_LIST=(
+    org.freedesktop.home1
+    org.freedesktop.hostname1
+    org.freedesktop.import1
+    org.freedesktop.locale1
+    org.freedesktop.login1
+    org.freedesktop.machine1
+    org.freedesktop.network1
+    org.freedesktop.oom1
+    org.freedesktop.portable1
+    org.freedesktop.resolve1
+    org.freedesktop.systemd1
+    org.freedesktop.timedate1
+    org.freedesktop.timesync1
+)
+
+SESSION_BUS_LIST=(
+    org.freedesktop.systemd1
+)
+
+# Configure dfuzzer suppressions to filter out interfaces which might degrade
+# or straight up kill the test machine
+cat >/etc/dfuzzer.conf <<EOF
+[org.freedesktop.login1]
+Halt destructive
+HaltWithFlags destructive
+Hibernate destructive
+HibernateWithFlags destructive
+HybridSleep destructive
+HybridSleepWithFlags destructive
+KillSession destructive
+KillUser destructive
+LockSession destructive
+LockSessions destructive
+PowerOff destructive
+PowerOffWithFlags destructive
+Reboot destructive
+RebootWithFlags destructive
+ReleaseSession destructive
+ScheduleShutdown destructive
+Suspend destructive
+SuspendThenHibernate destructive
+SuspendThenHibernateWithFlags destructive
+SuspendWithFlags destructive
+TerminateSeat destructive
+TerminateSession destructive
+TerminateUser destructive
+
+[org.freedesktop.timedate1]
+SetNTP destructive (disables systemd-timesyncd)
+EOF
+
+# Overmount /var/lib/machines with a size-limited tmpfs, as fuzzing
+# the org.freedesktop.machine1 stuff makes quite a mess
+mount -t tmpfs -o size=50M tmpfs /var/lib/machines
+
+# Fuzz both the system and the session buses (where applicable)
+for bus in "${BUS_LIST[@]}"; do
+    echo "Bus: $bus (system)"
+    # Activate the name (if it's activatable) before fuzzing it
+    busctl introspect "$bus" /
```

Comment:
Without `--no-pager` it seems in interactive mode it keeps asking to press "Enter"
```
Failed to execute 'pager', using next fallback pager: No such file or directory
WARNING: terminal is not fully functional
Press RETURN to continue
```

I also added `ASAN_OPTIONS=detect_leaks=0` to run it manually under ASan/UBSan.

Another issue I ran into is that it can't seem to activate `homed`
```
+ for bus in "${BUS_LIST[@]}"
+ echo 'Bus: org.freedesktop.home1 (system)'
Bus: org.freedesktop.home1 (system)
+ busctl introspect org.freedesktop.home1 /
Bus n/a: changing state UNSET → OPENING
sd-bus: starting bus by connecting to /run/dbus/system_bus_socket...
Bus n/a: changing state OPENING → AUTHENTICATING
Bus n/a: changing state AUTHENTICATING → HELLO
Sent message type=method_call sender=n/a destination=org.freedesktop.DBus path=/org/freedesktop/DBus interface=org.freedesktop.DBus member=Hello cookie=1 reply_cookie=0 signature=n/a error-name=n/a errora
Got message type=method_return sender=org.freedesktop.DBus destination=:1.37 path=n/a interface=n/a member=n/a cookie=4294967295 reply_cookie=1 signature=s error-name=n/a error-message=n/a
Bus n/a: changing state HELLO → RUNNING
Sent message type=method_call sender=n/a destination=org.freedesktop.home1 path=/ interface=org.freedesktop.DBus.Introspectable member=Introspect cookie=2 reply_cookie=0 signature=n/a error-name=n/a erroa
Got message type=error sender=org.freedesktop.DBus destination=:1.37 path=n/a interface=n/a member=n/a cookie=4294967295 reply_cookie=2 signature=s error-name=org.freedesktop.DBus.Error.NameHasNoOwner er.
Failed to introspect object / of service org.freedesktop.home1: Could not activate remote peer.
Bus n/a: changing state RUNNING → CLOSED
+ at_exit
```
so I had to start it manually as well.

Reply:
Interesting, it worked fine for me locally. Looks like this is going to be fun in CIs :-)

Reply:
@mrc0mmand I'll take a closer look later but as far as I can remember `dfuzzer` sometimes just stops fuzzing for no apparent reason and half the methods kind of stuck in limbo. I'm not sure whether timeouts/connections are configurable there but it would make sense to look at that as well.